bfsos/bucket_test.go

```diff
@@ -0,0 +1,58 @@
+package bfsos_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/bsm/bfs/bfsos"
+	"github.com/bsm/bfs/testdata/lint"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Bucket", func() {
+	var (
+		tmp  string
+		data lint.Data
+	)
+
+	BeforeEach(func() {
+		if os.Getenv("BFSOS_TEST") == "" {
```

Comment:
actaully, I don't think we need this. this test can run on any env, it doesn't need an env guard